<ui version="4.0" >
 <class>ITrackEditWidget</class>
 <widget class="QWidget" name="ITrackEditWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QToolButton" name="toolGraph" >
       <property name="toolTip" >
        <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'URW Gothic L'; font-size:14pt; font-weight:200; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Toggle track statistics view.&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolExit" >
       <property name="toolTip" >
        <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'URW Gothic L'; font-size:14pt; font-weight:200; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Close this view.&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="treePoints" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights" >
      <bool>true</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>#</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>time</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>altitude</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>delta</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>azimuth</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>distance</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>speed</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>position</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QComboBox" name="comboColor" />
     </item>
     <item>
      <widget class="QLineEdit" name="lineName" />
     </item>
     <item>
      <widget class="QCheckBox" name="checkRemoveDelTrkPt" >
       <property name="text" >
        <string>remove purged track points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkResetDelTrkPt" >
       <property name="text" >
        <string>reset purged track points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="standardButtons" >
        <set>QDialogButtonBox::Apply</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
